bundles/asm-policy-v0.0.1/1.1.2_asm-sidecar-injection.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: AsmSidecarInjection
+metadata:
+  name: asm-sidecar-injection
+  annotations:
+    description: Enforce the istio proxy sidecar always been injected to workload pods.
+    bundles.validator.forsetisecurity.org/asm-policy-v0.0.1: 1.1.2
+spec:
+  enforcementAction: dryrun
+  match:
+    kinds:
+      - apiGroups:
+          - ''
+        kinds:
+          - Pod
+    excludedNamespaces:
+      - istio-system
+  parameters:
+    strictnessLevel: High # kpt-set: ${strictness-level}
```

Comment:
Is there a reason this is capitalized? Is it case sensitive? It seems unusual.

Reply:
Ye, the template has enum with High or Low.

Reply:
Ack.